CONTRIBUTING.md

```diff
@@ -33,23 +32,23 @@ opened using [this form](https://github.com/galaxyproject/galaxy/issues/new).
 
 ## How to Contribute
 
-* All changes to the [core galaxy project](https://github.com/galaxyproject/galaxy)
-  should be made through pull requests to this repository (with just two
-  exceptions outlined below).
+* All changes to the [core galaxy
+  project](https://github.com/galaxyproject/galaxy) should be made through pull
+  requests to this repository (with just two exceptions outlined below).
 
-* If you are new to Git, the [Try Git](http://try.github.com/) tutorial is a good places to start.
-  More learning resources are listed at https://help.github.com/articles/good-resources-for-learning-git-and-github/ .
+* If you are new to Git, the [Try Git](http://try.github.com/) tutorial is a
+  good places to start.  More learning resources are listed at
+  https://help.github.com/articles/good-resources-for-learning-git-and-github/
 
 * Make sure you have a free [GitHub](https://github.com/) account.
 
 * Fork the [galaxy repository](https://github.com/galaxyproject/galaxy) on
-  GitHub to make your changes.
-  (While many Galaxy instances track
+  GitHub to make your changes.  (While many Galaxy instances track
   [galaxy-dist](https://bitbucket.org/galaxy/galaxy-dist), active development
```

Comment:
Should we drop this bit about bitbucket/galaxy/galaxy-dist at this point?

Reply:
Yes, please!